src/scss/grommet-core/_objects.drop.scss

```diff
@@ -3,9 +3,19 @@
 .drop {
   position: fixed;
   z-index: 20;
-  background-color: $background-color;
+  background-color: rgba(255, 255, 255, 0.9);
   border: $drop-border;
   border-radius: $border-radius;
   box-shadow: $drop-box-shadow;
   overflow: auto;
-}
+  
+  .menu__control--fixed-label {
```

Comment:
Since Drop could be used by component other than Menu, I would think these selectors belong in _objects.menu.scss.